compair/api/comparison_example.py

```diff
@@ -100,7 +105,9 @@ class ComparisonExampleRootAPI(Resource):
     def get(self, course_uuid, assignment_uuid):
         course = Course.get_active_by_uuid_or_404(course_uuid)
         assignment = Assignment.get_active_by_uuid_or_404(assignment_uuid)
-        require(READ, ComparisonExample(course_id=course.id))
+        require(READ, ComparisonExample(course_id=course.id),
+            title="Failed to Retrieve Comparison Examples",
+            message="You do not have permission to view comparison examples for this assignment since you are not the course's instructor.")
```

Comment:
"Comparison Example Unavailable" / "Your role in this course does not allow you to view comparison examples."